server/applicationInfo.ts

```diff
@@ -0,0 +1,12 @@
+import fs from 'fs'
+import path from 'path'
+import config from './config'
+
+const packageJson =
+  process.env.NODE_ENV !== 'test' ? path.join(__dirname, '../../package.json') : path.join(__dirname, '../package.json')
```

Comment:
not very happy with the check to see if we're running a test or not.

Problem is that ts-jest transpiles js on the fly and the running dir is the ts source folder, whereas when running the application we're one layer down (in the transpilied code in the dist folder)

Reply:
tidied this up in later commit